fonts.css

```diff
@@ -0,0 +1,40 @@
+@font-face {
+    font-family: 'pt_serifbold';
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/html-css-2-0/pullRequest?prid=6240894)